web/js/main.js

```diff
@@ -198,6 +198,12 @@ window.onload = () => {
           }
         });
       }
+      // handle legacy config categories
+      if (!config.categoryGroupOrder) {
```

Comment:
I think this is fine for this release but, there are lots of other config files that need to be present that we don't/wouldn't throw errors for in `main.js` if they were not present so this feels a little out of place.  I wonder if maybe a warning/error should be part of the config building process during which, if necessary, this file could actually be generated based on the categories found in the category configs.

Reply:
Ok, I opted to throw an error during the build process. The building of a json file using the category directories sounds like something interesting I'd like to see in a follow up ticket if you have an idea to handle that.

Reply:
I will take a quick stab at it this afternoon and see if I can include it in this PR.  If not, this is a good solution for now.

Reply:
Ok, I pushed a little script that will generate the `categoryGroupOrder.json` if it's not present.  Let me know what you think.

Reply:
Nice, I think that will work!